react/src/views/Transfers/CreateTransferAgreement/CreateTransferAgreementView.tsx

```diff
@@ -100,45 +100,28 @@ function CreateTransferAgreementView() {
       (base) => parseInt(base.value, 10),
     );
 
-    let sourceOrganisationId;
-    let targetOrganisationId;
-    let targetBaseIds;
-    let sourceBaseIds;
     let transferType: TransferAgreementType;
-
     switch (createTransferAgreementData.transferType) {
       case "Sending to":
-        sourceOrganisationId = globalPreferences?.selectedOrganisationId;
-        sourceBaseIds = currentOrgBaseIds;
-        targetOrganisationId = createTransferAgreementData.partnerOrganisation.value;
-        targetBaseIds = partnerBaseIds;
-        transferType = TransferAgreementType.Unidirectional;
+        transferType = TransferAgreementType.SendingTo;
```

Comment:
@vahidbazzaz it's still necessary to have the old assignment of source/target org/bases because I didn't change the API in that regard: the idea is still that one org acts as source, and one as target, instead of one being the "creating" and the other the "accepting" one. Sorry for not explicitly mentioning it in #623, I thought it was clear due to the small code change.

Reply:
yes, exactly

Reply:
As discussed, I revert my request and instead implement this logic in the BE, also updating the authorization.
In the meantime the "Receiving-From" agreements-creation will be broken though.

One final thing @vahidbazzaz can you merge master one more time (it now holds some FE refactoring changes by @HaGuesto and I want to be certain everything's still okay)

Reply:
@HaGuesto @vahidbazzaz here we agreed (and we just confirmed) that the source-target-switch logic does not happen in the FE but in the BE (implemented through #636 ).
Also I'll update the naming to talk about "initiating" and "partner" org when creating an agreement.